R/train.R

```diff
@@ -28,7 +31,7 @@
 #' learner = makeLearner("classif.rpart", minsplit = 7, predict.type = "prob")
 #' mod = train(learner, task, subset = training.set)
 #' print(mod)
-train = function(learner, task, subset = NULL, weights = NULL) {
+train = function(learner, task, subset = NULL, weights = NULL, cache = FALSE) {
```

Comment:
See above -- I would not add `cache` as an argument here.

Reply:
Same as above. I initially tried without adding `cache` as an arg but it seems we cannot go without.